docs/Breaking API Changes.md

```diff
@@ -46,3 +46,10 @@ The change affects compatibility in two ways:
 - DateTimeConstant(-1) will still count when we check that you don’t specify two default values. The compiler will produce an error, instead of succeeding (and producing IL with two attributes).
 
 PR: https://github.com/dotnet/roslyn/pull/11536
+
+# Version 4.1.0
```

Comment:
This document is mostly outdated. https://github.com/dotnet/roslyn/issues/54242 is tracking documenting API breaking changes. But no conclusion yet it seems.